test/integration_test/specs/kubevirt-fedora-vm/portworx/sc-sharedv4svc-nolock-remote-only.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+  name: sc-sharedv4svc-nolock-remote-only
+parameters:
+  repl: "3"
```

Comment:
Might be better to use "repl=2" here since it is more predicable for the tests especially when using multiple volumes. Or may be create multiple storage classes.

Reply:
fixed.